f2/src/locales/fr.json

```diff
@@ -14,7 +14,7 @@
   "analystReport.reportLanguage": "Langue du rapport",
   "analystReport.reportNumber": "Signaler le numéro",
   "analystReport.reportVersion": "Version du rapport",
-  "analystReport.selfHarmString": "mots d'automutilation détectés",
```

Comment:
Where did this translation come from?

Reply:
Coming from Harm Word in title already defined and used, and PR raiser wish this the same as title